tests/conftest.py

```diff
@@ -171,27 +167,16 @@ def test_ts():
 
 @pytest.fixture(scope="function")
 def test_iam_df():
-    if IamDataFrame is None:
-        pytest.skip("pyam is not installed")
+    if not IamDataFrame:
```

Comment:
oh right, we probably want to keep this to make sure we can at least initialise from an IamDataFrame...